### ISSUE TITLE
Windows 7: NVDA Loses Focus on Items when Alt-Tabbing Back to Windows Explorer Lists

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#7086]
This is a new issue that follows on from #6881 - apologies for the delay on testing this!

Unfortunately, after updating to the latest Next snapshot (13994,744a2d39) I'm still experiencing the same issue as described previously. Do you have any more ideas as to what the cause could be? I'm obviously more than happy to carry out testing.

**Comment by zahra21** [comment]
hello.
can someone please tell me how can i disable aero if i can test this issue on windows 7?
is it possible that problem be the regression of fixing bug
https://github.com/nvaccess/nvda/issues/6671
did you test this issue with nvda 2016.4 which does not have this fix?
https://www.nvaccess.org/files/nvda/releases/2016.4/nvda_2016.4.exe
thanks for any advice, God bless you!

**Comment by ehollig** [comment]
@elliott94 can you still replicate this issue?

**Comment by elliott94** [comment]
I haven't seen this on higher-end hardware for a while - closing for now.
